src/js/directives/directives.js

```diff
@@ -871,46 +871,48 @@ module.directive('rpSpanSpacing', [function () {
 }]);
 
 /**
- * My Orders widget header.
+ * Used on header for my Orders widget and Contacts list.
  */
-module.directive('rpOrdersSortHeader', ['$timeout', '$parse', function($timeout, $parse) {
+module.directive('rpSortHeader', ['$timeout', '$parse', function($timeout, $parse) {
   return {
     restrict: 'A',
     compile: function (element, attr, linker) {
       return function (scope, element, attr) {
-        if (!attr.rpOrdersSortHeaderField) {
+        if (!attr.rpSortHeaderField) {
           // no field specified, do nothing
           return;
         }
-        var sortFieldGetter = $parse(attr.rpOrdersSortHeader);
-        var sortReverse = $parse(attr.rpOrdersSortHeaderReverse);
-        var fieldName = attr.rpOrdersSortHeaderField;
+        var sortFieldGetter = $parse(attr.rpSortHeader);
+        var sortReverse = $parse(attr.rpSortHeaderReverse);
+        var fieldName = attr.rpSortHeaderField;
+
+        function setArrowClass(sorted, isUp) {
+          var i = element.find('i');
+          i.toggleClass('sorted', sorted);
+          i.toggleClass(isUp ? 'fa-caret-up' : 'fa-caret-down');
```

Comment:
@rht this will end up with div having both 'fa-caret-up' and 'fa-caret-down' set